cni/network/network_test.go

```diff
@@ -1186,7 +1186,7 @@ func TestPluginSwiftV2Add(t *testing.T) {
 				IfName:      eth0IfName,
 			},
 			wantErr:    true,
-			wantErrMsg: "IPAM Invoker Add failed with error: delegatedVMNIC fail",
+			wantErrMsg: "IPAM Invoker Add failed with error: nodeNetworkInterfaceFrontendNIC fail",
```

Comment:
NIT: can we add comments somewhere to explain the meaning of "frondendNIC" and "backendNIC"?

Reply:
In `NetworkContainerContract.go`, meaning of `NodeNetworkInterfaceFrontendNIC `was documented under `DelegatedVMNIC`